boskos/client/client.go

```diff
@@ -93,6 +94,25 @@ func (c *Client) Acquire(rtype, state, dest string) (*common.Resource, error) {
 	return r, nil
 }
 
+// AcquireWait blocks until Acquire returns the specified resource or the
+// provided context is cancelled or its deadline exceeded.
+func (c *Client) AcquireWait(ctx context.Context, rtype, state, dest string) (*common.Resource, error) {
+	// Try to acquire the resource until available or the context is
+	// cancelled or its deadline exceeded.
+	for {
+		r, err := c.Acquire(rtype, state, dest)
```

Comment:
you might want to sleep before retries, as you could DOS the boskos server.

Reply:
I'm literally working on that now. When I saw the pushed commit I realized the issue :) Thanks!